solver.py

```diff
@@ -30,7 +32,7 @@ def name(self):
 
     
     def iterate(self, mdp, gamma, V_0):
-        V = V_0
+        V = copy.deepcopy(V_0)
```

Comment:
@JordanTroutman I guess this is the bug you were talking about, not sure if you fixed it in the same way

Reply:
Yep this was the bug